### PR TITLE
replace openid-configuration well-known URI with jwt issuer well-known URI

### PR DESCRIPTION
**Pull request by Sakurann**
issue #127

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/148.html" title="Last updated on Sep 12, 2023, 5:04 PM UTC (a5acb3e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/148/1247eed...a5acb3e.html" title="Last updated on Sep 12, 2023, 5:04 PM UTC (a5acb3e)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -86,6 +86,13 @@
           status:   "Internet-Draft",
           publisher:  "IETF"
         },
+        "SD-JWT VC": {
```
Comment (OR13) [inline index.html:89]:
```suggestion
        "SD-JWT-VC": {
```

File: index.html [hunk 2]
```diff
@@ -462,99 +469,17 @@ <h2>Well Known URIs</h2>
...
-      
       <section>
-        <h2>OpenID Connect</h2>
+        <h2>JWT Issuer</h2>
             <p>
-              OpenID Connect uses <a data-cite="RFC5785#section-3">Well-Known Uniform Resource Identifiers (URIs)</a> 
-              to enable <a data-cite="VC-DATA-MODEL#dfn-issuers">issuer</a> key discovery.
+              When the issuer value is a URL using the HTTPS scheme,
+              issuer metadata including the issuer's public keys can be retrieved using the mechanism
+              defined in <a data-cite="SD-JWT VC"></a>.
```
Comment (OR13) [inline index.html:477]:
```suggestion
              defined in <a data-cite="SD-JWT-VC"></a>.
```

File: index.html (hunk 1 above)
Comment (selfissued) [inline index.html:89]: I don't think you can have a space in a reference tag.  Maybe change the space to dash.

**Review by selfissued — APPROVED** [review]
Works for me.

**Review by brentzundel — APPROVED** [review]

**Pull request merged**